src/components/time-picker/time-picker.tsx

```diff
@@ -34,6 +35,11 @@ const ampmSwitch = {
     [Ampm.NONE]: Ampm.NONE
 };
 const segments: Segment[] = ['hh', 'mm', 'ampm'];
+const helpLabels = {
+    hh: 'Hours',
+    mm: 'Minutes',
+    ampm: 'AM/PM'
+};
```

Comment:
I generally think that good idea is to store all strings separately, it will be easier then to find them. 
put it in separate file, lets say strings.js and use it from there, I will write proposal to make it through all components.

Reply:
i will try to find solution without using any strings

Reply:
created `strings.ts`